docs/cmd/kn_service_create.md

```diff
@@ -39,26 +39,27 @@ kn service create NAME --image IMAGE [flags]
-      --wait-timeout int         Seconds to wait before giving up on waiting for service to be ready. (default 60)
+      --async                         Create service and don't wait for it to become ready.
+      --concurrency-limit int         Hard Limit of concurrent requests to be processed by a single replica.
+      --concurrency-target int        Recommendation for when to scale up based on the concurrent number of incoming request. Defaults to --concurrency-limit when given.
+  -e, --env stringArray               Environment variable to set. NAME=value; you may provide this flag any number of times to set multiple environment variables. To unset, specify the environment variable name followed by a "-" (e.g., NAME-).
+      --force                         Create service forcefully, replaces existing service if any.
+  -h, --help                          help for create
+      --image string                  Image to run.
+  -l, --label stringArray             Service label to set. name=value; you may provide this flag any number of times to set multiple labels. To unset, specify the label name followed by a "-" (e.g., name-).
+      --limits-cpu string             The limits on the requested CPU (e.g., 1000m).
+      --limits-memory string          The limits on the requested memory (e.g., 1024Mi).
+      --lock-to-digest                keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision) (default true)
+      --max-scale int                 Maximal number of replicas.
+      --min-scale int                 Minimal number of replicas.
+  -n, --namespace string              List the requested object(s) in given namespace.
+      --no-lock-to-digest             do not keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision)
+  -p, --port int32                    The port where application listens on.
+      --requests-cpu string           The requested CPU (e.g., 250m).
+      --requests-memory string        The requested memory (e.g., 64Mi).
+      --revision-name string          The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
+      --service-account-name string   Service account name. To unset, specify "-". (default "-")
```

Comment:
Let’s make clearer with “To unset specify “-“ as the value”

Also would this be clearer with “” instead of “-“?

Reply:
I agree. I will make the change.

Reply:
How can you unset something when you create a service ? Isn't it just good enough to leave out the option if you don't want to create a service account being set ?

Reply:
Oh, I see. I'll make it optional.

Reply:
I've changed the description of the flag with the change of the name of the flag.